jenkinsfiles/android_jenkinsfile.groovy

```diff
@@ -1,13 +1,31 @@
 /**
 * Android Jenkinsfile
 */
-node('android') {
-    stage 'Checkout'
+node("android"){
+  stage("checkout"){
     checkout scm
+  }
 
-    stage 'Build'
-    sh "./gradlew clean assembleDebug"
+  stage("build"){
+    sh 'chmod +x ./gradlew'
+    sh './gradlew clean assembleRelease' // builds app/build/outputs/app-release-unsigned.apk file
+  }
 
-    stage 'Archive'
-    archiveArtifacts artifacts: 'app/build/outputs/apk/*.apk', excludes: 'app/build/outputs/apk/*-unaligned.apk'
-}
+  stage("sign"){
```

Comment:
@wei-lee I took this from [here](https://github.com/wei-lee/welcome-android-gradle/blob/master/Jenkinsfile) as suggested by @odra Is there a more updated version of this Jenkinsfile that we can use?